tests/test_annotations.py

```diff
@@ -130,15 +132,15 @@ def test_correct_url(self):
             # The destination file will be missing.
             with pytest.raises(FileNotFoundError):
                 an.download()
-        mocked_get.assert_called_once_with(an.zarr_url, filename=mock.ANY)
+        mocked_get.assert_called_once_with(an.intervals_url, filename=mock.ANY)
 
     def test_incorrect_url(self):
         an = AnnotationTestClass()
-        an.zarr_url = "http://asdfwersdf.com/foozip"
+        an.intervals_url = "http://asdfwersdf.com/foozip"
         with pytest.raises(OSError):
             an.download()
 
-    @pytest.mark.xfail  # HomSap annotation not currently available
+    # @pytest.mark.xfail  # HomSap annotation not currently available
```

Comment:
If the test now passes, the xfail decorator can be removed. Ditto for test below.

Reply:
done